### PR TITLE
Docs: Fix nightly builds link

### PR DESCRIPTION
**Pull request by imatwawana**
Replace dead link for nightly OSS builds

### CODE REVIEW

File: docs/sources/_index.md [hunk 1]
```diff
@@ -44,7 +44,7 @@ title: Grafana documentation
         </div>
         <h5>Grafana Cloud</h5>
     </a>
-    <a href="https://grafana.com/grafana/nightly?edition=oss" class="nav-cards__item nav-cards__item--install">
+    <a href="https://grafana.com/grafana/download?edition=oss" class="nav-cards__item nav-cards__item--install">
```
Comment (imatwawana) [inline docs/sources/_index.md:47]:
```suggestion
    <a href="https://grafana.com/grafana/download/10.1.0-125833pre?pg=get&plcmt=selfmanaged-box1-cta1&edition=oss" class="nav-cards__item nav-cards__item--install">
```

File: docs/sources/_index.md [hunk 2]
```diff
@@ -44,7 +44,7 @@ title: Grafana documentation
         </div>
         <h5>Grafana Cloud</h5>
     </a>
-    <a href="https://grafana.com/grafana/nightly?edition=oss" class="nav-cards__item nav-cards__item--install">
+    <a href="https://grafana.com/grafana/download/10.1.0-125833pre?pg=get&plcmt=selfmanaged-box1-cta1&edition=oss" class="nav-cards__item nav-cards__item--install">
```
Comment (dsotirakis) [inline docs/sources/_index.md:47]: Hm, this points specifically to the `10.1.0-125833pre` Grafana version 🤔 
Maybe someone needs to actually fix this URL.

Comment (imatwawana) [reply docs/sources/_index.md:47]: @dsotirakis Yeah, I was asking that question in the Slack thread; do we have to update this every minor release? It seems like whatever that old link was, it wasn't release specific at least.

Comment (dsotirakis) [reply docs/sources/_index.md:47]: Nope, it's even worse we need to update it nightly 😅 
I think we need to escalate this with someone (don't know with whom though 😞) to actually probably create a `https://grafana.com/grafana/nightly` url for us.

Comment (imatwawana) [reply docs/sources/_index.md:47]: @dsotirakis - I haven't gotten any response in Slack so I'll open an issue on the repo. It doesn't look like this nightly build has been updated nightly for a while (it's the same build today as it was when I opened this PR—so maybe these builds aren't being posted?). For now, I think we should just point to the existing nightly build and make a more permanent fix later.

Comment (imatwawana) [inline docs/sources/_index.md:47]:
```suggestion
    <a href="https://grafana.com/grafana/download/nightly?edition=oss">
```

File: docs/sources/_index.md [hunk 3]
```diff
@@ -44,7 +44,7 @@ title: Grafana documentation
         </div>
         <h5>Grafana Cloud</h5>
     </a>
-    <a href="https://grafana.com/grafana/nightly?edition=oss" class="nav-cards__item nav-cards__item--install">
+    <a href="https://grafana.com/grafana/download/nightly?edition=oss">
```
Comment (imatwawana) [inline docs/sources/_index.md:47]:
```suggestion
    <a href="https://grafana.com/grafana/download/nightly?edition=oss" class="nav-cards__item nav-cards__item--install">
```

**Review by dsotirakis — APPROVED** [review]
This looks good! 🚀

**Pull request merged**